plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyJob.java

```diff
@@ -157,28 +121,19 @@ public void run() {
             try {
                 shell.run(script,name, new String[] {});
                 String outputStream = shell.getProperty("out").toString();
+                GroovyConsoleContent.writeFileNameIntoConsole(name);
                 GroovyConsoleContent.writeIntoConsole(outputStream, true);
                 GroovyConsoleContent.writeIntoConsole("END");
                 status = IStatus.OK;
-                closeOutputPrintWriter();
-            } catch (MissingPropertyException e){
-                e.printStackTrace(pw);
-                String sStackTrace = sw.toString();
-                GroovyConsoleContent.writeIntoConsole(sStackTrace);
-                LocalDateTime now = LocalDateTime.now();
-                outputPrintWriter.write("\n" + dtf.format(now) + "\n" + sStackTrace);
-                GroovyConsoleContent.writeIntoConsole("BAD_END");   
-                status = IStatus.ERROR;
-                closeOutputPrintWriter();
-            } catch(Exception e){
+            } catch (Exception e){
                 e.printStackTrace(pw);
                 String sStackTrace = sw.toString();
+            	GroovyConsoleContent.writeFileNameIntoConsole(name);
                 GroovyConsoleContent.writeIntoConsole(sStackTrace);
-                LocalDateTime now = LocalDateTime.now();
-                outputPrintWriter.write("\n" + dtf.format(now) + "\n" + sStackTrace);
-                GroovyConsoleContent.writeIntoConsole("BAD_END");   
+                GroovyConsoleContent.writeIntoConsole("BAD_END");
```

Comment:
It seems that this PR is not up to date with the master
https://github.com/orbisgis/orbisrcp/blob/master/plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyJob.java#L136